cmake/CompilerHelper.cmake

```diff
@@ -5,3 +5,22 @@ if("${CMAKE_CXX_COMPILER_ID}" MATCHES "GNU")
     message(FATAL_ERROR "${PROJECT_NAME} requires g++ >= 4.9 (for c++11 support)")
   endif()
 endif()
+
+if(CMAKE_CXX_COMPILER_ID MATCHES "PGI")
+  # CMake adds standard complaint PGI flag "-A" which breaks compilation of of spdlog and fmt
+  set(CMAKE_CXX11_STANDARD_COMPILE_OPTION --c++11)
+  set(CMAKE_CXX14_STANDARD_COMPILE_OPTION --c++14)
+
+  # PGI with llvm code generation doesn't have necessary assembly intrinsic headers
+  add_compile_definitions(EIGEN_DONT_VECTORIZE=1)
```

Comment:
I guess it's ok, but I find this an astonishing place to put this command - I would move this into the main `CMakeLists.txt` to where we discover the Eigen package.

Reply:
yeah agree!